Cargo.toml

```diff
@@ -11,13 +11,15 @@ categories = ["embedded", "no-std", "hardware-support"]
 readme = "README.md"
 
 [features]
-default = [ "lan8720a", "lan8742a", "ksz8081r" ]
+default = ["lan8720a", "lan8742a", "ksz8081r", "dp83640", "dp83848"]
```

Comment:
Unless we can test the additions, we should not have these features activated by default. Fine for now, but if we merge without actually ever running the tests we should remember to change this.

Reply:
You're right, I think I'll leave this PR here until I fix my board or someone else test these out for us. Thanks for the comments! I'm still new to embedded Rust and this has been an amazing journey for me :)

Reply:
I will change the PR to a draft until that happens, then :)